workflows/rnaseq/main.nf

```diff
@@ -787,6 +787,385 @@ workflow RNASEQ {
+        from QUANTIFY_RSEM.out.merged_tpm_transcript
+    }
+
+    // subworkflows/nf-core/bam_markduplicates_picard
+    "${params.aligner}" {
+        'picard_metrics' {
+            from BAM_MARKDUPLICATES_PICARD.out.metrics
+        }
+        from BAM_MARKDUPLICATES_PICARD.out.bam
+        from BAM_MARKDUPLICATES_PICARD.out.bai
+        from BAM_MARKDUPLICATES_PICARD.out.csi
+        'samtools_stats' {
+            from BAM_MARKDUPLICATES_PICARD.out.stats
+            from BAM_MARKDUPLICATES_PICARD.out.flagstat
+            from BAM_MARKDUPLICATES_PICARD.out.idxstats
+        }
+    }
+
+    // subworkflows/nf-core/bam_rseqc
+    "${params.aligner}/rseqc" {
+        'bam_stat' {
+            from BAM_RSEQC.out.bamstat_txt
+        }
+        'infer_experiment' {
+            from BAM_RSEQC.out.inferexperiment_txt
+        }
+        'junction_annotation' {
+            'pdf' {
+                from BAM_RSEQC.out.junctionannotation_pdf
+                from BAM_RSEQC.out.junctionannotation_events_pdf
+            }
+            'bed' {
+                from BAM_RSEQC.out.junctionannotation_bed
+                from BAM_RSEQC.out.junctionannotation_interact_bed
+            }
+            'xls' {
+                from BAM_RSEQC.out.junctionannotation_xls
+            }
+            'log' {
+                from BAM_RSEQC.out.junctionannotation_log
+            }
+            'rscript' {
+                from BAM_RSEQC.out.junctionannotation_rscript
+            }
+        }
+        'junction_saturation' {
+            'pdf' {
+                from BAM_RSEQC.out.junctionsaturation_pdf
+            }
+            'rscript' {
+                from BAM_RSEQC.out.junctionsaturation_rscript
+            }
+        }
+        'read_duplication' {
+            'pdf' {
+                from BAM_RSEQC.out.readduplication_pdf
+            }
+            'xls' {
+                from BAM_RSEQC.out.readduplication_seq_xls
+                from BAM_RSEQC.out.readduplication_pos_xls
+            }
+            'rscript' {
+                from BAM_RSEQC.out.readduplication_rscript
+            }
+        }
+        'read_distribution' {
+            from BAM_RSEQC.out.readdistribution_txt
+        }
+        'inner_distance' {
+            'txt' {
+                from BAM_RSEQC.out.innerdistance_distance
+                from BAM_RSEQC.out.innerdistance_freq
+                from BAM_RSEQC.out.innerdistance_mean
+            }
+            'pdf' {
+                from BAM_RSEQC.out.innerdistance_pdf
+            }
+            'rscript' {
+                from BAM_RSEQC.out.innerdistance_rscript
+            }
+        }
+        'tin' {
+            from BAM_RSEQC.out.tin_txt
+        }
+    }
+
+    // subworkflows/nf-core/fastq_align_hisat2
+    "${params.aligner}" {
+        'log' {
+            from FASTQ_ALIGN_HISAT2.out.summary
+        }
+        from FASTQ_ALIGN_HISAT2.out.orig_bam, enabled: params.save_align_intermeds
```

Comment:
`enabled` should be a group attribute (if really needed)

Reply:
that would not be specific enough, in most cases only specific outputs are made optional based on a param, not entire directories